addons/mail/static/src/models/tracking_value.js

```diff
@@ -0,0 +1,55 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { attr, one } from '@mail/model/model_field';
+import { sprintf } from '@web/core/utils/strings';
+
+registerModel({
+    name: 'TrackingValue',
+    identifyingFields: ['id'],
+    recordMethods: {
+        /**
+         * @private
+         * @returns {string}
+         */
+        _computeChangedFieldAsString() {
+            return sprintf(this.env._t("%s:"), this.changedField);
+        },
+    },
+    fields: {
+        /**
+         * Indicate the original field of changed tracking value, such as "Status", "Date".
+         */
+        changedField: attr({
+            required: true,
+        }),
+        /**
+         * The formatted string of the changed field name, such as "Status", "Date", etc.
+         */
+        changedFieldAsString: attr({
+            compute: '_computeChangedFieldAsString',
+        }),
```

Comment:
This comment is not helpful, it should tell what is the difference between "changedField" and "changedFieldAsString" instead of repeating the other comment.

Also I don't like this field name too much. It's literally just `changedFieldAndSemicolon`?